### DISCUSSION TITLE
cannot find using_geodata.py

### DISCUSSION THREAD

**Discussion by JaneNamesZhou** [mozman/ezdxf#816] [Q&A]
cannot find using_geodata.py

**Comment by mozman** [comment]
This example script has been renamed to [examples/entities/geodata_setup_local_grid.py](https://github.com/mozman/ezdxf/blob/master/examples/entities/geodata_setup_local_grid.py)

**Reply by JaneNamesZhou** [reply]
one more question: if i know the coord of dxf file is EPSG:XXXX, how can i set to its geo?

**Reply by mozman** [reply]
I wrote the documentation and tutorial for the `geo` add-on so I don't have to remember these things since I don't use georeferencing. If you can't find the answer in the documentation, you're on your own.

**Reply by JaneNamesZhou** [reply]
Thanks for your reply~
